protobuf/src/test/java/io/grpc/protobuf/ProtoUtilsTest.java

```diff
@@ -60,6 +62,17 @@ public void testJsonRoundtrip() throws Exception {
     assertEquals(proto, marshaller.parse(is));
   }
 
+  @Test
+  public void testJsonMarshallerFailToPrint() {
+    Marshaller<Any> marshaller = ProtoUtils.jsonMarshaller(Any.getDefaultInstance());
+    try {
+      marshaller.stream(Any.newBuilder().setValue(ByteString.copyFromUtf8("invalid")).build());
+      fail("Expected exception");
+    } catch (StatusRuntimeException e) {
+      assertEquals(Status.Code.INTERNAL, e.getStatus().getCode());
```

Comment:
This can also check that the cause is present: `assertNotNull(e.getCause());

Reply:
Added.